src/main/java/com/nhnacademy/marketgg/server/entity/Category.java

```diff
@@ -13,33 +13,72 @@
 import javax.persistence.ManyToOne;
 import javax.persistence.Table;
 
+/**
+ * 카테고리 Entity 입니다.
+ *
+ * @version 1.0.0
+ */
 @Table(name = "categories")
 @Entity
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
 @Getter
 public class Category {
 
+    /**
+     * 카테고리의 식별코드 입니다.
+     *
+     * @since 1.0.0
+     */
     @Id
     @Column(name = "category_code")
     private String categoryCode;
 
+    /**
+     * 카테고리가 속할 카테고리 분류 입니다.
+     *
+     * @since 1.0.0
+     */
     @ManyToOne
     @JoinColumn(name = "categorization_code")
     private Categorization categorization;
 
+    /**
+     * 카테고리의 이름입니다.
+     *
+     * @since 1.0.0
```

Comment:
모든 속성 마다 Java Docs를 다셨네요.
모든 엔티티 속성 마다 Java Docs를 달까요 아니면 Optional하게 남겨두는 것이 좋을까요?
개인적으로 Optional에 한 표를 던지고 싶습니다.

```
/**
 * Specifies that the class is an entity. This annotation is applied to the
 * entity class.
 *
 * @since 1.0
 */
@Documented
@Target(TYPE)
@Retention(RUNTIME)
public @interface Entity {

  /**
   * (Optional) The entity name. Defaults to the unqualified
   * name of the entity class. This name is used to refer to the
   * entity in queries. The name must not be a reserved literal
   * in the Jakarta Persistence query language.
   */
  String name() default "";
}
```

이와 같이 꼭 필요한 속성에만 Java Docs를 다는것이 좋아 보입니다.

Reply:
좋은 의견입니다. 팀 컨벤션 회의 때 말해보겠습니다!